viz.Rmd

```diff
@@ -671,35 +1012,67 @@ vector graphics as much as you like without the image looking bad, while raster
+[@faithfuldata].
 
 ```{r 03-plot-line, collapse=TRUE, warning=FALSE, message=FALSE, fig.cap = "Scatter plot of waiting time and eruption time"}
 library(svglite) # we need this to save SVG files
 faithful_plot <- ggplot(data = faithful, aes(x = waiting, y = eruptions)) +
-  geom_point()
+  geom_point() +
+  labs(x = "Waiting time to next eruption \n (minutes)",
+       y = "Eruption time \n (minutes)")
 
 faithful_plot
+```
 
+Now that we have a named `ggplot` plot object, we can use the `ggsave` function
+to save a file containing this image. 
+`ggsave` works by taking a file name to create for the image 
+as it's first argument. 
+This can include the path to the directory where you would like to save the file 
+(e.g., `img/filename.png` to save a file named `filename` to the `img` directory),
+and the name of the plot object to save as it's second argument.
+The kind of image to save is specified by the file name's file extension.
+For example, 
+to create a PNG image file we specify that the file extension is `.png`.
+Below we demonstrate how to save PNG, JPG, BMP, TIFF and SVG file types 
+for the `faithful_plot`:
+
+```{r warning=FALSE, message=FALSE}
 ggsave("img/faithful_plot.png", faithful_plot)
 ggsave("img/faithful_plot.jpg", faithful_plot)
 ggsave("img/faithful_plot.bmp", faithful_plot)
 ggsave("img/faithful_plot.tiff", faithful_plot)
 ggsave("img/faithful_plot.svg", faithful_plot)
+```
 
-print(paste("PNG filesize: ", file.info("img/faithful_plot.png")["size"] / 1000000, "MB"))
-print(paste("JPG filesize: ", file.info("img/faithful_plot.jpg")["size"] / 1000000, "MB"))
-print(paste("BMP filesize: ", file.info("img/faithful_plot.bmp")["size"] / 1000000, "MB"))
-print(paste("TIFF filesize: ", file.info("img/faithful_plot.tiff")["size"] / 1000000, "MB"))
-print(paste("SVG filesize: ", file.info("img/faithful_plot.svg")["size"] / 1000000, "MB"))
+```{r, filesizes, echo = FALSE}
+file_sizes <- tibble(`Image type` = c("Bitmap / Raster", 
+                        "Bitmap / Raster", 
+                        "Bitmap / Raster", 
+                        "Bitmap / Raster",
+                        "Vector / Scalable Graphics"),
+       `File type` = c("PNG", "JPG", "BMP", "TIFF", "SVG"),
+       `Image size` = c(paste(round(file.info("img/faithful_plot.png")["size"] / 1000000, 2), "MB"),
+                        paste(round(file.info("img/faithful_plot.jpg")["size"] / 1000000, 2), "MB"),
+                        paste(round(file.info("img/faithful_plot.bmp")["size"] / 1000000, 2), "MB"),
+                        paste(round(file.info("img/faithful_plot.tiff")["size"] / 1000000, 2), "MB"),
+                        paste(round(file.info("img/faithful_plot.svg")["size"] / 1000000, 2), "MB")))
+kable(file_sizes,
+      caption = "File sizes of `faithful_plot` when saved as different file formats.")
 ```
+
+The different file types are not only different in their properties 
```

Comment:
formatting in this section (writing for the rest of the chapter centered rather than left aligned and the caption formatting displays in the pdf) 
<img width="602" alt="Screen Shot 2021-10-01 at 1 31 48 PM" src="https://user-images.githubusercontent.com/3838091/135682882-53efbbd4-b1bf-48cd-88b7-fd01e657a4a9.png">

Reply:
OK, this is weird and should not happen, I will check tables in the formatting pass.